examples/deploy-heroku/manifest.json

```diff
@@ -0,0 +1,29 @@
+{
+  "display_information": {
+    "name": "Bolt on Heroku Example"
+  },
+  "features": {
+    "bot_user": {
+      "display_name": "Bolt on Heroku Example",
+      "always_online": true
```

Comment:
Related to the `eco` dyno that can sleep - I think it's alright to suppose the bot is always online. It just might be slow to response 🤔